packages/helper-image/test/unit/spec/image-orientation-util.js

```diff
@@ -0,0 +1,93 @@
+/**!
+*
+* Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+*/
+
+import {assert} from '@ciscospark/test-helper-chai';
+import * as ImageOrientationUtil from '../..';
```

Comment:
Have you run this test? I'm pretty sure `import *` doesn't work with the current (reasonably conservative) babel config.